NAMESPACE

```diff
@@ -7,3 +7,4 @@ import(Rcpp)
 import(methods)
 importFrom(magrittr,"%>%")
 importFrom(rstan,sampling)
+useDynLib(bsynth, .registration = TRUE)
```

Comment:
google3 has problems when this line is present.  I do not know why.

roxygen automatically produces it.  I removed it by hand.

I do not know what the best approach is.

Reply:
removing this line breaks things outside google. We will use copybara or something like that to remove it for google when importing it into third_party